src/components/Tabs.jsx

```diff
@@ -7,28 +7,48 @@ import classNames from 'classnames'
 class Tabs extends React.Component {
     constructor(props, context) {
         super(props, context)
-        this.state = { activeTab: props.defaultTab || 0 }
+
+        const { defaultTab, onChange } = props
+        const children = React.Children.toArray(this.props.children)
+
+        if (defaultTab || onChange) {
+            const missing = children.find(c => !c.props.value)
+            if (missing) throw new Error('(Tab) Missing property: all Tab must have "value" set if "defaultTab" or "onChange" is used')
+        }
+
+        if (defaultTab) {
+            const c = children.find((x) => x.props.value === defaultTab)
```

Comment:
nitpick: This can be done in one call with `children.findIndex(...)`

Reply:
I think `findIndex` require some polyfill. Does Reactist in general requires use of polyfill?

Reply:
I think babel is taking care of that, right? You shouldn't need to import any other polyfills

Reply:
Hmm I think for new object types and prototype methods, you will need to include polyfill explicitly; the same way you need polyfill if you use Promise.

Anyway, I don't think it is a major issue since both Todoist/Twist already use polyfill anyway 🙂